src/city_limit_finder.py

```diff
@@ -0,0 +1,24 @@
+import overpy
+api = overpy.Overpass()
+
+quick_c = [43.947300096957186, -79.7710501044725, 43.58421719106671, -78.98435889455229]
+#quick_c is quick coordinate, paste in two coordinates into the list above and it will be re-ordered for OSM as the bounding box
```

Comment:
I see that you are swapping values here so quick_c[0] > quick_c[2] and quick_c[1] > quick_c[3], but what is the point of this file if we have to input the coords ourself?